vllm/v1/engine/output_processor.py

```diff
@@ -53,11 +56,13 @@ def from_new_request(
         cls,
         tokenizer: AnyTokenizer,
         request: EngineCoreRequest,
+        lora_request: Optional[LoRARequest],
```

Comment:
`EngineCorerRequest` already has `LoRARequest` as a member (https://github.com/vllm-project/vllm/blob/33170081f13b4e4453f0fff8df994df8d9aa6c6c/vllm/v1/engine/__init__.py#L61) so you do not need to pass the `lora_request` as an argument to this function